packages/conf-libelf/conf-libelf.1.0.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+authors: ["Bayge <alex@fluidity.money>"]
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+build: ["pkg-config" "--exists" "libelf"]
+depexts: [
+  ["libelf-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["libelf"] {os-distribution = "arch"}
+  ["dev-libs/libelf"] {os-distribution = "gentoo"}
+  ["libelf"] {os = "freebsd"}
+  ["libelf"] {os = "macos" & os-distribution = "homebrew"}
+  ["libelf-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["libelf"] {os = "alpine"}
```

Comment:
```suggestion
  ["elfutils-dev"] {os = "alpine"}
```
The Alpine workflows are failing. Searching pkgs.org I found `elfutils-dev` contains the relevant `/usr/lib/pkgconfig/libelf.pc` that `pkg-config` will search for:
https://alpine.pkgs.org/3.21/alpine-main-aarch64/elfutils-dev-0.191-r0.apk.html